webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationService.java

```diff
@@ -161,40 +148,35 @@ public DefaultCsvValidationService(final SchoolRepository schoolRepository) {
                 studentCountPerGroup++;
             }
         }
+
         validateStudentCount(failures, currentSchool, currentGroup, studentCountPerGroup, recordNumber);
 
-        if(failures.isEmpty()) {
-            successes.add(newSuccessValidationResult(recordNumber, groupCountPerSchool, getSchoolOrganization(availableSchools, currentSchool)));
+        if (failures.isEmpty()) {
+            successes.add(success(recordNumber, groupCountPerSchool, getSchool(availableSchools, currentSchool)));
             return ImmutableList.copyOf(successes);
         }
 
         return ImmutableList.copyOf(failures);
     }
 
-    private void validateStudentCount(final List<CsvValidationResult> validationFailures,
+    private void validateStudentCount(final List<CsvValidationResult> failures,
                                       final String school,
                                       final String group,
                                       final int studentCount,
                                       final long recordNumber) {
-        if (studentCount < MaxStudentsPerGroup) return;
-
-        validationFailures.add(CsvValidationResult.builder()
-                .row(recordNumber)
-                .message("School [" + school + "] Group [" + group + "] contains " + studentCount + " students.  Maximum students allowed per group is " + MaxStudentsPerGroup)
-                .isOk(false)
-                .build());
+        if (studentCount > MaxStudentsPerGroup) {
```

Comment:
The actual bug fix

Reply:
I like most of the other changes you made ... made things more readable. But why change away from guard-code style here? Just to clarify, if the code remained guard-code style, the fix would be `<` -> `<=`?

Reply:
I'm good with removing the guard-code style here in favor of consistency with the other validations.

Reply:
@malaffoon Yes ```<=``` would have fixed the bug.
Also, I changed away from the guard to be consistent with the rest of the methods like @bgsavage said.